sdk/attestation/attestation/README.md

```diff
@@ -274,18 +317,29 @@ Additional information on how to perform attestation token validation can be fou
 
 ### Retrieve Token Certificates
 
-Use `get_signing_certificates` to retrieve the certificates which can be used to validate the token returned from the attestation service.
+Use `getSigningCertificates` to retrieve the certificates which can be used to validate the token returned from the attestation service.
 
 ```ts
-<FILL THIS IN>
+const client = new AttestationClient(new DefaultAzureCredential(), endpoint);
+
+const attestationSigners = await client.getAttestationSigners();
+
+console.log(`There are ${attestationSigners.length} signers`);
+
 ```
 
 ## Troubleshooting
 
-Most Attestation service operations will raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/README.md). The attestation service APIs will throw a `HttpResponseError` on failure with helpful error codes. Many of these errors are recoverable.
+Most Attestation service operations will raise exceptions defined in [Azure Core](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/core/README.md). The attestation service APIs will throw a `RestError` on failure with helpful error codes. Many of these errors are recoverable.
 
 ```ts
-<Fill this in>
+try {
+  await client.attestSgxEnclave(_openEnclaveReport, {
+    runTimeJson: stringToBytes('{"bogus": 10 }')
+  });
+} catch (error) {
+  console.log(`Expected Exception thrown for invalid request: ${error.message}`);
+}
 ```
```

Comment:
can we update the links at the end of the file to point to the recent version or to be version agonistic (not sure how tbh)? For example there is `[api_reference]: https://azuresdkdocs.blob.core.windows.net/$web/javascript/azure-attestation/1.0.0-beta.2/index.html` which points to v1.0.0-beta.2 but most recent version is v1.0.0-beta.4.

Reply:
Perhaps use this instead for api reference: https://docs.microsoft.com/en-us/javascript/api/@azure/attestation

Reply:
Yeah, I was wondering about that... Updating.

Btw the 1.0.0-beta.4 version has a ton of stuff that isn't in the index.ts, so I'm not sure how it showed up in the docs.

Reply:
Yes you should use the link I just posted because it points to the official docs. The link you have is not the official docs. @ramya-rao-a have you seen packages pointing to the azuresdkdocs website before?

Reply:
I think you should use the official and the version-agnostic website: https://docs.microsoft.com/en-us/javascript/api/@azure/attestation

Reply:
I see many references to https://azuresdkdocs.blob.core.windows.net/ and I think we should replace all of them with more appropriate sources. Please take a look at text analytics's readme for reference: https://raw.githubusercontent.com/Azure/azure-sdk-for-js/main/sdk/textanalytics/ai-text-analytics/README.md

Reply:
I switched over to use the docs.microsoft.com version, instead of azuresdkdocs. 

docs.microsoft.com is a complete disaster, IMHO but at least it has a version agnostic, language agnostic version of the documentation.